src/core_ocean/shared/mpas_ocn_tracer_hmix_redi.F

```diff
@@ -216,12 +218,13 @@ subroutine ocn_tracer_hmix_Redi_tend(meshPool, scratchPool, layerThicknessEdge,
 
                ! For top layer, only use triads pointing down:
                flux_term2 = coef * RediKappa(iCell) * layerThicknessEdge(k, iEdge) * &
-                  0.5_RKIND * &
+                  0.25_RKIND * &
```

Comment:
I think we need to treat the top and bottom as an average of 4 things even though there are just two.  I think of it as an average of 4 triads still but 2 are zero.